tests/v1/kv_connector/nixl_integration/toy_proxy_server.py

```diff
@@ -151,18 +151,8 @@ async def send_request_to_service(client_info: dict, endpoint: str,
     Send a request to a service using a client from the pool.
```

Comment:
I pulled this from the old nm/disagg_pd_dev branch. The one on was triggering some TP communication issues between workers.

Reply:
let's hold until I figure this out